### ISSUE TITLE
Add term Specialized extracellular matrix

### ISSUE THREAD

**Issue by DaiqingChen** [geneontology/go-ontology#29265]
### Suggested term label:
Specialized extracellular matrix

### Definition (free text)
Species-specific or cell-type specific extracellular matrices that are different from the two main types of extracellular matrices: the interstitial ECM (GO:0005614) and the basement membrane ECM (GO:0005604).

### Gene product name and ID to be annotated to this term
Keep the original product name and ID

### Parent term(s)
Extracellular matrix (GO:0031012)

### Children terms 
* **Terms that were originally under Extracellular matrix (GO:0031012)** 

- [x] Pollen coat (GO:0070505)
- [x] Ubisch body (GO:0070645)
- [x] Egg coat (GO:0035805)
- [x] Chitin-based extracellular matrix (GO:0062129)
- [x] Middle lamella (GO:0009519)
- [x] Biofilm matrix (GO:0062039)
- [x] Organomineral extracellular matrix (GO:1990377)
- [ ] Drosophila adhesive extracellular matrix (#29499 )

* **Terms that were originally under collagen-containing extracellular matrix (GO:0062023)** 

- [x] Synapse-associated extracellular matrix (GO:0099535)
- [x] Interphotoreceptor matrix (GO:0033165)
- [x] Cuticular ECM (GO:0060102)
- [x] Hyaline layer (GO:0033166)

- [x] Note: We are also requesting changing the name of term GO:0060102 from "collagen and cuticulin-based cuticle extracellular matrix" to "Cuticular ECM"

### Synonyms (please specify, EXACT, BROAD, NARROW or RELATED)
Not applicable

**See slide 2**: https://docs.google.com/presentation/d/1zxL0gt5vOynmJsS8BP-Xqs_0sTaEB8Zo/edit?usp=sharing&ouid=109267399050361021326&rtpof=true&sd=true

Slide screenshot: 
<img width="1518" alt="Image" src="https://github.com/user-attachments/assets/d11b8173-b124-450e-94a2-a50a838be945" />

**Comment by pgaudet** [comment]
@DaiqingChen 
Ubisch body (GO:0070645) is currently part_of ECM. I am making it part_of specialized ECM. OK ?

**Comment by pgaudet** [comment]
@DaiqingChen 
For terms that that were originally under collagen-containing extracellular matrix (GO:0062023): 
-  Synapse-associated extracellular matrix (GO:0099535)
-  interphotoreceptor matrix (GO:0033165)
-  Cuticular ECM (GO:0060102)

I have ADDED 'specialized extracellular matrix' as an additional parent; did you want to remove/obsolete collagen-containing extracellular matrix (GO:0062023) ? I dont see this term in Framework from slide 2: https://docs.google.com/presentation/d/1CfY-qBrZLK-7TBR8GB9V87uYdBdPmeg9/edit?usp=sharing&ouid=109267399050361021326&rtpof=true&sd=true


Thanks, Pascale

**Comment by pgaudet** [comment]
New term: 
+[Term]
+id: GO:0140047
+name: specialized extracellular matrix
+namespace: cellular_component
+def: "Species or cell-type specific extracellular matrices that are different from the two main types of extracellular matrices: the interstitial ECM and the basement membrane ECM in metazoa." [PMID:33605520]
+synonym: "specialized ECM" EXACT []
+is_a: GO:0031012 ! extracellular matrix
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29265" xsd:anyURI
+created_by: pg
+creation_date: 2024-12-13T08:19:31Z

**Comment by pgaudet** [comment]
@DaiqingChen 
We need one or more references for GO terms. I used PMID:33605520, let me know if there are more appropriate ones.

**Comment by DaiqingChen** [comment]
> [@DaiqingChen](https://github.com/DaiqingChen) For terms that that were originally under collagen-containing extracellular matrix (GO:0062023):
> 
> * Synapse-associated extracellular matrix (GO:0099535)
> * interphotoreceptor matrix (GO:0033165)
> * Cuticular ECM (GO:0060102)
> 
> I have ADDED 'specialized extracellular matrix' as an additional parent; did you want to remove/obsolete collagen-containing extracellular matrix (GO:0062023) ? I dont see this term in Framework from slide 2: https://docs.google.com/presentation/d/1CfY-qBrZLK-7TBR8GB9V87uYdBdPmeg9/edit?usp=sharing&ouid=109267399050361021326&rtpof=true&sd=true
> 
> Thanks, Pascale

Yes, I would like to obsolete collagen-containing extracellular matrix (GO:0062023). Haven't got the chance to request obsolete term yet, will write it up soon, thank you Pascale!

**Comment by DaiqingChen** [comment]
> [@DaiqingChen](https://github.com/DaiqingChen) Ubisch body (GO:0070645) is currently part_of ECM. I am making it part_of specialized ECM. OK ?

I would think it should be is_a ECM, as only plants has this specialized ECM type. A reference about it: https://doi.org/10.1016/0034-6667(94)00086-Y
@Matrisome what do you think?

**Comment by DaiqingChen** [comment]
> [@DaiqingChen](https://github.com/DaiqingChen) We need one or more references for GO terms. I used PMID:33605520, let me know if there are more appropriate ones.

I would think we should cite reference more focused on plants(PMID: 28955324) and bacteria(PMID: 28089324) for specialized ECM, also SynGO for the synapse-related terms(PMID: 31171447)

**Comment by pgaudet** [comment]
Thanks @DaiqingChen 

I added the references, except for the SynGo paper, which doesn't describe the ECM. 

Thanks, Pascale

**Comment by DaiqingChen** [comment]
Hi @Matrisome, 
     The term adhesive extracellular matrix(GO:0062130) we originally wanted to change from Cellular Component to Molecular Function, but after reading the definition, this term is defining a matrix in drosophila where "These glands synthesize and secrete massive amounts of a glue which attaches the pupae to the substrate during metamorphosis." 
      Looks like this term should be added under specialized ECM, and update the name to the synonym "puparial glue". What do you think?

**Comment by Matrisome** [comment]
@DaiqingChen : please update with screenshot of revised slide 2

**Comment by pgaudet** [comment]
In fact GO:0062130  adhesive extracellular matrix was obsoleted in #29499
